src/zenml/integrations/neptune/__init__.py

```diff
@@ -29,9 +29,7 @@ class NeptuneIntegration(Integration):
     """Definition of the neptune.ai integration with ZenML."""
 
     NAME = NEPTUNE
-    REQUIREMENTS = [
-        "neptune-client>=0.16.10",
-    ]
+    REQUIREMENTS = ["neptune"]
```

Comment:
do we want / need to pin this to some lower version?

Reply:
Like even should it be >=1.0 at least?

Reply:
1.0 is the smallest version that exists for `neptune` :)

Reply:
Would we then consider that a breaking change?

Reply:
Since people would be using <1.0 with `neptune-client`